src/components/ItemList/data.tsx

```diff
@@ -0,0 +1,63 @@
+const BEERS_PER_PAGE: number = 10;
+const DEFAULT_PAGE: number = 1;
+const ROOT_ENDPOINT: string = 'https://api.punkapi.com/v2/';
+const INITIAL_ENDPOINT: string = `beers?page=${DEFAULT_PAGE}&per_page=${BEERS_PER_PAGE}`;
+const BEER_NAME: string = '&beer_name=';

```

Comment:
Could  you keep global variables in one file or folder

Reply:
https://github.com/ylakhtin/rs-react-components/commit/5f4a7d6fe4fea2271c6d9148c8e939e4cd5c4946